src/System.Private.ServiceModel/tests/Scenarios/Security/TransportSecurity/Https/ClientCredentialTypeTests.cs

```diff
@@ -24,20 +27,18 @@ static Https_ClientCredentialTypeTests()
 #if FULLXUNIT_NOTSUPPORTED
     [Fact]
 #else
-    [ConditionalFact(nameof(Root_Certificate_Installed), nameof(Basic_Authentication_Available))]
+    [ConditionalFact(nameof(Root_Certificate_Installed))]
 #endif
     [OuterLoop]
     public static void BasicAuthentication_RoundTrips_Echo()
```

Comment:
There were two tests for basic auth, and one of them is a negative test and also is passing.